.github/workflows/test.yml

```diff
@@ -63,14 +63,15 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ['3.9', '3.10', '3.11', '3.12']
+        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
 
     steps:
     - uses: actions/checkout@v4
     - uses: actions/setup-python@v5
       name: Install Python
       with:
-        python-version: '3.11'
+        python-version: ${{ matrix.python-version }}
```

Comment:
Can't believe we weren't actually using this before 🤦

Reply:
At least the conda tests didn't have this bug, so it's not like we were never testing all Python versions.

Reply:
Nice catch! Amazing that this one has slipped by, taking a look at the former build.yml this has persisted a few years.